gemfiles/without_rails.gemfile

```diff
@@ -8,5 +8,6 @@ gem "pg", platform: :ruby
 gem "pry"
 gem "pry-stack_explorer", platform: :ruby
 gem "activerecord-jdbcsqlite3-adapter", platform: :jruby
+gem "globalid"
```

Comment:
This file is generated from the config in `Appraisals`, could you also update that file?

Reply:
Updated.